Composer/packages/client/src/components/AddRemoteSkillModal/CreateSkillModal.tsx

```diff
@@ -73,7 +79,46 @@ export const validateManifestUrl = ({ formData, formDataErrors, setFormDataError
   }
 };
 
-export const getSkillManifest = async (projectId: string, manifestUrl: string, setSkillManifest, setFormDataErrors) => {
+export const validateLocalZip = async (files: Record<string, JSZipObject>) => {
+  const result: { error: any; zipContent?: Record<string, string>; manifestContent?: any; path: string } = {
+    error: {},
+    path: '',
+  };
+  try {
+    // get manifest
+    const manifestFiles: JSZipObject[] = [];
+    const zipContent: Record<string, string> = {};
+    for (const fPath in files) {
+      zipContent[fPath] = await files[fPath].async('string');
+      // eslint-disable-next-line no-useless-escape
+      if (fPath.match(/\.([^\.]+)$/)?.[1] === 'json' && isManifestJson(zipContent[fPath])) {
+        manifestFiles.push(files[fPath]);
+        result.path = fPath.substr(0, fPath.lastIndexOf('/') + 1);
+      }
+    }
+
+    // update content for detail panel and show it
+    if (manifestFiles.length > 1) {
```

Comment:
The .zip file will also contain the .lu file.

Reply:
@EricDahlvang Here we only check if manifest.json is existed and valid, for lu files, we will check it later and show a warning if it is not accessible like the following, which means, if there is no lu files in .zip, user can still add the manifest but they need to configure their own trigger phrases. 

![image](https://user-images.githubusercontent.com/32191031/124109292-8fa8fc00-da99-11eb-831f-726080edf420.png)